app/src/main/java/com/alexbar/layoutbasic/ui/theme/Type.kt

```diff
@@ -3,52 +3,75 @@ package com.alexbar.layoutbasic.ui.theme
 import androidx.compose.material3.Typography
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.Font
 import androidx.compose.ui.text.font.FontFamily
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.unit.sp
+import com.alexbar.layoutbasic.R
+
+val poppinsFamily = FontFamily(
+    Font(R.font.poppins_thin),
+    Font(R.font.poppins_light),
+    Font(R.font.poppins_regular),
+    Font(R.font.poppins_medium),
+    Font(R.font.poppins_bold),
+    Font(R.font.poppins_black),
+    Font(R.font.poppins_extra_bold),
+    Font(R.font.poppins_semi_bold),
+)
 
 // Set of Material typography styles to start with
 val Typography = Typography(
     bodyLarge = TextStyle(
-        fontFamily = FontFamily.Default,
+        fontFamily = poppinsFamily,
```

Comment:
@ezatpanah  To add a custom font, do I only need to make these changes, or is there an additional step I need to take?

Reply:
Yes , the way that you used , it is correct 👍🏼